NitroxPatcher/Patches/Dynamic/DayNightCycle_OnConsoleCommand_daynightspeed_Patch.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public class DayNightCycle_OnConsoleCommand_daynightspeed_Patch : NitroxPatch, IDynamicPatch
+{
+    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((DayNightCycle t) => t.OnConsoleCommand_daynightspeed(default));
+
+    // The command is skipped because simulating speed reliable on the server is out of scope
+    public static bool Prefix()
+    {
+#if DEBUG
+        ErrorMessage.AddMessage("This command is not fully supported. Expect breaking your instance/world.");
+        return true;
+#else
+        ErrorMessage.AddMessage(Language.main.Get("Nitrox_CommandNotAvailable"));
+        return false;
+#endif
```

Comment:
Do we need to enable them in DEBUG ?

Reply:
We don't need it right now. Will change it, if somebody needs the command we devs can disable it manually.